frontend/src/scenes/onboarding/OnboardingInviteTeammates.tsx

```diff
@@ -49,7 +50,12 @@ export const OnboardingInviteTeammates = ({ stepKey }: { stepKey: OnboardingStep
             title={`${titlePrefix()} better with friends.`}
             stepKey={stepKey}
             hedgehog={<PhonePairHogs height={120} width={288} />}
-            continueAction={() => preflight?.email_service_available && inviteTeamMembers()}
+            continueAction={() =>
+                preflight?.email_service_available &&
+                invitesToSend[0]?.target_email &&
```

Comment:
it is strict, but it doesn't prevent them from continuing if it's false. If it's true, then we can send the invites, otherwise we don't pass a `continueAction` and it just completes onboarding.